pontoon/sync/changeset.py

```diff
@@ -147,6 +147,12 @@ def update_entity_translations_from_vcs(
             )
         approved_translations = []
 
+        # Unfuzzy existing fuzzy translations to prevent duplicate fuzzies.
+        for translation in db_translations:
```

Comment:
So I tested this, and it seems to be working without a problem:
`bulk_update([Translation.objects.get(pk=13813), Translation.objects.get(pk=13813)])`

Still, at the bottom of the function I could do something like:
`self.translations_to_update = list(set(self.translations_to_update))`